README.md

```diff
@@ -238,7 +220,69 @@ the [Maven version range format](https://cwiki.apache.org/confluence/display/MAV
 | (,1.0],[1.2,) | x <= 1.0 or x >= 1.2. Multiple sets are comma-separated                       |
 | (,1.1),(1.1,) | This excludes 1.1 if it is known not to work in combination with this library |
 
-#### External Dependencies: Runs
+#### Local Files
+
+You can also include files built by other tasks in your project, for example, jar tasks of other source sets.
+
+When wanting to build a secondary jar for a coremod or service, you could define a separate source set `service`,
+add a jar task to package it and then include the output of that jar like this:
+
+```groovy
+sourceSets {
+    service
+}
+
+
+neoForge {
+    // ...
+    mods {
+        // ...
+        // To make the service load in dev
+        'service' {
+            sourceSet sourceSets.service
+        }
+    }
+}
+
+def serviceJar = tasks.register("serviceJar", Jar) {
+    from(sourceSets.service.output)
+    manifest.attributes["FMLModType"] = "LIBRARY"
+    archiveClassifier = "service"
+    manifest {
+        attributes(
+                'FMLModType': "LIBRARY",
+                "Automatic-Module-Name": project.name + "-service"
+        )
+    }
+}
+
+dependencies {
+    jarJar files(serviceJar)
+}
+```
+
+When you include a jar file like this, we use its filename as the artifact-id and its MD5 hash as the version. 
+It will never be swapped out with embedded libraries of the same name, unless their content matches.
+
+#### Subprojects
+
+For example, if you have a coremod in a subproject and want to embed its jar file, you can use the following syntax.
+
+```groovy
+dependencies {
+    jarJar project(":coremod")
+}
+```
+
+When starting the game, FML will use the group and artifact id of an embedded Jar-file to determine if the same file
+has been embedded in other mods.
+For subprojects, the group id is the root project name, while the artifact id is the name of the subproject.
```

Comment:
...what no this is such unexpected behavior. The group ID should be the group ID of the subproject, obviously. Which is the default behavior for cross-project dependencies in general. Since the subproject dependency is a "real" one, with a proper version name and all, people should be specifying a group for their subproject...

Reply:
What is the default, e.g. if you don't specify a group ID in the subproject?

Reply:
Project path, with dots as separators. This is why the MDK _ought_ to have `group=com.example.examplemod` in gradle.properties instead of whatever it's doing with `mod_group_id`, so that it applies to subprojects. I mean, there's some other issues there too, but whatever -- my point on this is just that as documented, it's just... _not_ how jarJar works.